src/kem/kyber/arithmetic.rs

```diff
@@ -62,29 +72,40 @@ pub(crate) fn montgomery_reduce(value: KyberFieldElement) -> KyberFieldElement {
     value_high - c
 }
 
+pub(crate) fn montgomery_multiply_sfe_by_fer(
```

Comment:
Let's inline these. They are probably inlined anyway, but let's make sure to tell the compiler.